<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreciseDrawDialog</class>
 <widget class="QDialog" name="PreciseDrawDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>矢量图精确绘制</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>264</width>
     <height>202</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="shapeTypeLabel">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>图形种类：</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="shapeTypeComboBox">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <item>
         <property name="text">
          <string>直线</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>矩形</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>矩形(可填充)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>标准圆</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>椭圆</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>三角形</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>曲线(贝塞尔)</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>确定绘图坐标与相应点位置：</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>开始点：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>X:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="startXSpinBox">
             <property name="maximum">
              <number>1024</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Y:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="startYSpinBox">
             <property name="maximum">
              <number>768</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>终止点：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>X:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="endXSpinBox">
             <property name="maximum">
              <number>1024</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Y:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="endYSpinBox">
             <property name="maximum">
              <number>768</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>控制点1：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>X:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="cAXSpinBox">
             <property name="maximum">
              <number>1024</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Y:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="cAYSpinBox">
             <property name="maximum">
              <number>768</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>控制点2：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>X:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="cBXSpinBox">
             <property name="maximum">
              <number>1024</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Y:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="cBYSpinBox">
             <property name="maximum">
              <number>768</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="cancelPushButton">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>PreciseDrawDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>PreciseDrawDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
